### PR TITLE
magick: update resizing example

### PR DESCRIPTION
**Pull request by kadang**
Update according to example at https://imagemagick.org/Usage/resize/

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- `magick --version`
Version: ImageMagick 7.1.1-18 Q16-HDRI x86_64 21582 https://imagemagick.org

### CODE REVIEW

File: pages/common/magick.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 - Resize an image, making a new copy:
 
-`magick convert -resize {{100x100}} {{image.jpg}} {{image.jpg}}`
+`magick {{image1.jpg}} -resize {{100x100}} {{image2.jpg}}`
```
Comment (acuteenvy) [inline pages/common/magick.md:13]:
```suggestion
`magick {{input_image.jpg}} -resize {{100x100}} {{output_image.jpg}}`
```

**Review by acuteenvy — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]
Approve with @acuteenvy 's suggestion

File: pages/common/magick.md (hunk 1 above)
Comment (kbdharun) [reply pages/common/magick.md:13]: In addition to this, I think we can use the path placeholder across the page for image files.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**